scripts/DeployTransmuter.s.sol

```diff
@@ -64,6 +66,10 @@ contract DeployTransmuter is Utils {
         facetNames.push("DiamondCut");
         facetAddressList.push(address(new DiamondCut()));
 
+        // TODO don't forget to set the dummy implementation as an implementation (after having regenerated it)
+        facetNames.push("DiamondEtherscan");
+        facetAddressList.push(address(new DiamondEtherscan()));
+
```

Comment:
Can you put back the Redeemer deployment at the very end as this one has trouble being verified on etherscan and the last time it stopped verifiying all the others